### PR TITLE
Add connect with params test, simpler connect string creation

### PR DESCRIPTION
**Pull request by JuanjoArreola**
simpler connect string creation from params.

### CODE REVIEW

**Comment by natebird** [comment]
Thanks for this. It looks good. I'm in the middle of updating the PostgreSQL stack to 2.0 Alpha and this will probably get merged into that.

File: Sources/PostgreSQL/Connection.swift [hunk 1]
```diff
@@ -22,17 +22,11 @@ public final class Connection: ConnInfoInitializable {
...
         case .params(let params):
-            var ci = ""
-            
-            params.forEach { (key, value) in
-                ci += "\(key)='\(value)'"
-            }
-        
-            string = ci
-        case .basic(host: let host, port: let port, database: let database, user: let user, password: let password):
+            string = params.map({ (key, value) in "\(key)='\(value)'" }).reduce("", +)
```
Comment (vzsg) [inline Sources/PostgreSQL/Connection.swift]: Why not just use `params.map({ (key, value) in "\(key)='\(value)'" }).joined()`?

Comment (JuanjoArreola) [reply Sources/PostgreSQL/Connection.swift]: Both look good to me, I can change it if thats preferred. Would it be too much to write it like this: `params.map({ "\($0)='\($1)'" }).joined()` ?

Comment (natebird) [reply Sources/PostgreSQL/Connection.swift]: I'm fine with the shorter syntax. It is seems readable enough to me.

Comment (JuanjoArreola) [reply Sources/PostgreSQL/Connection.swift]: Alright, I'll update the pull request

**Pull request merged**